Dockerfile

```diff
@@ -174,6 +173,9 @@ COPY utils/bin /root/.local/bin
 # Setup requirements for cluster-login.sh and install o-must-gather
 RUN pip3 install requests-html o-must-gather && pyppeteer-install
 
+# Setup pagerduty-cli
+RUN npm install -g pagerduty-cli
```

Comment:
We should probably lock this down to a specific version or use a package.json

Reply:
Added tag for specific version: `RUN npm install -q pagerduty-cli@0.0.70`